README.md

```diff
@@ -25,9 +25,9 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Shayan Portfolio] <a name="about-project"></a>
+# 📖 [First Capston Project] <a name="about-project"></a>
 
-> This website is for a car development company named Worrior
+> Castop project is my first near real-work project experience in which I set up a new Github repo, added files and configured code linters before ensuring that the git/github flows worked as expected. Then I proceeded to style the content focusing on a mobile-first approach, based on an original design idea by Cindy Shin in Behance.
```

Comment:
- Kindly, would you add the designer of this design in the `acknowledgment` section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).👍👍